pkgs/development/python-modules/altgraph/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "altgraph";
+  version = "0.17.3";
+
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ad33358114df7c9416cdb8fa1eaa5852166c505118717021c6a8c7c7abbd03dd";
+  };
+
+  doCheck = false;
+
+  meta = with lib; {
+    changelog = "https://github.com/ronaldoussoren/altgraph/tags${version}";
+    description = "A fork of graphlib: a graph (network) package for constructing graphs";
+    longDescription = ''
+      altgraph is a fork of graphlib: a graph (network) package for constructing graphs,
+      BFS and DFS traversals, topological sort, shortest paths, etc. with graphviz output.
+      altgraph includes some additional usage of Python 2.6+ features and enhancements related to modulegraph and macholib.
+    '';
+    homepage = "https://altgraph.readthedocs.io/";
+    downloadPage = "https://pypi.org/project/altgraph/";
```

Comment:
Wouldn't it be better to just set the pypi.org url as the homepage and drop the link to readthedocs.io?

Reply:
Thanks for the review!

Isn't that what these tags are for? I mean, that is what's listed as Homepage on pypi. Don't see harm in adding that here as well.

But don't have a steong opinion on that. If you feel like thats better I can do that.

Reply:
My thinking was that the pypi link would be more useful for the homepage link in the nixpkgs search, (and I found other python modules use the pypi link as homepage link, the 3 examples I found of it as downloadPage used the github repo as homepage), but I don't have very strong feelings about the matter.

Reply:
I thinks it's fine this way.